spec/overlap/multi_part_overlap_spec.rb

```diff
@@ -95,10 +100,33 @@
       expect(overlap.matching_holdings.count).to eq(1)
     end
 
-    it "does not find holdings with the wrong enum" do
```

Comment:
The previous test had one htitem and a holding with a different enumchron from the holding. That now *matches* (to avoid matches, we need to have *multiple* holdings, at least one of which matches a *different* htitem in the same cluster/record)